src/navigator/IFrameNavigator.ts

```diff
@@ -2667,18 +2650,7 @@ export default class IFrameNavigator implements Navigator {
           }
         }
       }
-
-      var widthRatio =
-        (parseInt(getComputedStyle(iframeParent).width) - 100) /
-        (this.iframes.length == 2
-          ? parseInt(width.replace("px", "")) * 2 + 200
-          : parseInt(width.replace("px", "")));
-      var heightRatio =
-        (parseInt(getComputedStyle(iframeParent).height) - 100) /
-        parseInt(height.replace("px", ""));
-      var scale = Math.min(widthRatio, heightRatio);
-      iframeParent.style.transform = "scale(" + scale + ")";
```

Comment:
It would be worth trying to find some other epubs with a fixed layout that are not these three graphic novel isbns, but the transform property seemed to me to be the main culprit of the weird cropping/scaling that was occuring with these isbns. I'm not sure in what scenario it would be useful because it was making the pages really really small....